packages/plugin-seerr/openapi-schema.yaml

```
@@ -305,6 +477,65 @@ components:
+        serverID:
+          type: string
+          readOnly: true
+    MetadataSettings:
+      type: object
+      properties:
+        settings:
+          type: object
+          properties:
+            tv:
+              type: string
+              enum: [ tvdb, tmdb ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)